uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/leave-request-data.js

```diff
@@ -1,101 +1,207 @@
-define(function () {
+define([
+  'mocks/data/option-group-mock-data',
+  'mocks/data/absence-type-data',
+], function (optionGroupMock, absenceTypeData) {
   var mockData = {
     allData: {
-      "is_error": 0,
-      "version": 3,
-      "count": 6,
-      "values": [
-        {
-          "id": "17",
-          "type_id": "1",
-          "contact_id": "202",
-          "status_id": "1",
-          "from_date": "2016-02-01",
-          "from_date_type": "1",
-          "to_date": "2016-02-03",
-          "to_date_type": "1"
-        },
-        {
-          "id": "18",
-          "type_id": "1",
-          "contact_id": "202",
-          "status_id": "1",
-          "from_date": "2016-08-17",
-          "from_date_type": "1",
-          "to_date": "2016-08-25",
-          "to_date_type": "1"
-        },
-        {
-          "id": "19",
-          "type_id": "1",
-          "contact_id": "202",
-          "status_id": "6",
-          "from_date": "2016-01-30",
-          "from_date_type": "1",
-          "to_date": "2016-02-01",
-          "to_date_type": "1"
-        },
-        {
-          "id": "20",
-          "type_id": "1",
-          "contact_id": "202",
-          "status_id": "3",
-          "from_date": "2016-11-23",
-          "from_date_type": "1",
-          "to_date": "2016-11-28",
-          "to_date_type": "1"
-        },
-        {
-          "id": "21",
-          "type_id": "3",
-          "contact_id": "202",
-          "status_id": "5",
-          "from_date": "2016-06-03",
-          "from_date_type": "1",
-          "to_date": "2016-06-13",
-          "to_date_type": "1"
-        },
-        {
-          "id": "22",
-          "type_id": "2",
-          "contact_id": "202",
-          "status_id": "4",
-          "from_date": "2016-01-01",
-          "from_date_type": "1",
-          "to_date": "2016-01-01",
-          "to_date_type": "1"
-        }
-      ]
+      'is_error': 0,
+      'version': 3,
+      'count': 6,
+      'values': [{
+        'id': '17',
+        'type_id': '1',
+        'contact_id': '202',
+        'status_id': '1',
+        'from_date': '2016-02-01',
+        'from_date_type': '1',
+        'to_date': '2016-02-03',
+        'to_date_type': '1'
+      }, {
+        'id': '18',
+        'type_id': '1',
+        'contact_id': '202',
+        'status_id': '1',
+        'from_date': '2016-08-17',
+        'from_date_type': '1',
```

Comment:
this is for array of leave requests coming from server, no point mocking them as they will correspond to real data and not some random combination of types